pyam/_aggregate.py

```diff
@@ -57,21 +57,36 @@ def _aggregate(df, variable, components=None, method=np.sum):
     return _group_and_agg(_df, [], method)
 
 
-def _aggregate_recursive(df, variable):
+def _aggregate_recursive(df, variable, recursive):
     """Recursive aggregation along the variable tree"""
 
     # downselect to components of `variable`, initialize list for aggregated (new) data
     _df = df.filter(variable=f"{variable}|*")
+    # keep variable at highest level if it exists
+    _df.append(df.filter(variable=variable), inplace=True)
```

Comment:
I thought about that, too. The problem is, I might have variables starting with the same words like 'Secondary Energy', 'Secondary Energy Input' and 'Secondary Energy Losses'. Filtering `df.filter(variable="Secondary Energy*")` would take the three of them instead of just 'Secondary Energy' as intended.

Reply:
Good point! I updated the suggestion above to take care of that (by changing it to a list of the `variable` and its components.